hbase-migration-tools/bigtable-hbase-replication/bigtable-hbase-replication-core/src/test/java/com/google/cloud/bigtable/hbase/replication/adapters/IncompatibleMutationAdapterTest.java

```diff
@@ -187,7 +194,10 @@ public void testDeleteCanCreateManyDeletes() {
 
     verify(metricsExporter).incCounters(INCOMPATIBLE_MUTATION_METRIC_KEY, 0);
     verify(metricsExporter, times(1)).incCounters(INCOMPATIBLE_MUTATION_METRIC_KEY, 1);
+    verify(metricsExporter, times(1)).incCounters(INCOMPATIBLE_MUTATION_DELETES_METRICS_KEY, 1);
     verify(metricsExporter, times(1)).incCounters(DROPPED_INCOMPATIBLE_MUTATION_METRIC_KEY, 0);
+    verify(metricsExporter, times(0))
```

Comment:
You can just call NoMoreInteractions on metrics exporter rather than calling expect(0) for all metrics.

Reply:
https://www.logicbig.com/tutorials/unit-testing/mockito/verify-no-more-interactions.html.